var/spack/repos/builtin/packages/py-hostlist/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHostlist(PythonPackage):
+    """Hostlist expands and collects LLNL hostlists."""
+
+    pypi = "python-hostlist/python-hostlist-1.23.0.tar.gz"
+    git = "https://github.com/LLNL/py-hostlist.git"
+
+    version("master", branch="master")
+    version(
+        "1.23",
+        sha256="56e0156b501f792c078114f07324f34f37827041581ee5d1ffdce89cca533219",
+        url="https://www.nsc.liu.se/~kent/python-hostlist/python-hostlist-1.23.tar.gz",
```

Comment:
This URL appears to fail and does not match the `pypi` setting at line 12.

When I run `spack checksum py-hostlist@1.23` I get `All fetchers failed...` error.

Reply:
I also get `Not Found` when I try the URL in a web browser.

Reply:
You can actually get rid of the `url` argument here and change the version to `1.23.0` at line 17.  Then `spack checksum py-hostlist@1.23.0` gets the provided sha256.

Reply:
Hi @tldahlgren, thanks for your review. 

This was my bad, I forgot the minor version in the url link.. It should be fixed (the `checksum` command works now).